README.rst

```diff
@@ -41,8 +41,8 @@ To install::
     $ python setup.py develop
 
 
-Setting up your project
-=======================
+Setting up your project by the defaults
```

Comment:
Meaning you name your folder examples and the gallery auto examples. And that you don't have to set the sphinx-gallery configuration dictionary. So the defaults